.github/workflows/pr-akka-samples.yml

```diff
@@ -88,17 +89,24 @@ jobs:
           cd akka-samples-${{ matrix.sample }}
           git config user.name "lightbend-tools-secure"
           git config user.email "lightbend-tools-secure@lightbend.com"
-          BRANCH=update-sdk-$sdk_version
+          # include timestamp to make the branch name unique
+          timestamp=`date '+%Y%m%d-%H%M%S'`
+          BRANCH=update-sdk-$sdk_version-$timestamp
           git checkout -b $BRANCH
           rm -rf ./src/*
           rsync -a ../docs/build/bundle/${{ matrix.sample }}/ ./
           rm -f .bundleignore
-          git add -f .idea/runConfigurations/Run_locally.xml
-          git add .
-          git commit . -m "chore: bump SDK version to $sdk_version"
-          git push --set-upstream origin $BRANCH
-          gh pr create -B main -t "Auto PR - Bump SDK version to $sdk_version" \
+          
+          if [ -n "$(git status --porcelain)" ]; then
```

Comment:
We are not running the script to update the version, instead we are taking the content from the zip bundle. 

It should contain the right version, but if we see that there is no changes, than it means that the bundle is wrong. Maybe we are looking at the wrong place and the issue is in the bundle generation.